README.md

```diff
@@ -1,9 +1,11 @@
 # Vehicle Damage Modeling
 
-This repository contains template projects for Vehicle Damage Modeling project.
+This repository contains Vehicle Damage Modeling project.
 
 ## iOS
 
+iOS project uses SceneKit - a 3D graphics application programming interface (API) developed by Apple Inc. 
```

Comment:
Please highlight 'SceneKit' the same as below is highlighted libGdX

Reply:
+1